### PR TITLE
kubeadm: Switching to apps/v1beta2

### PR DESCRIPTION
**Pull request by sbezverk**
Closes https://github.com/kubernetes/kubeadm/issues/390

### CODE REVIEW

File: cmd/kubeadm/app/phases/selfhosting/selfhosting.go [hunk 1]
```diff
@@ -22,8 +22,8 @@ import (
 	"os"
 	"time"
 
+	extensions "k8s.io/api/apps/v1beta2"
```
Comment (luxas) [inline cmd/kubeadm/app/phases/selfhosting/selfhosting.go]: Should be apps now

File: cmd/kubeadm/app/phases/upgrade/health.go [hunk 2]
```diff
@@ -21,8 +21,8 @@ import (
 	"net/http"
 	"os"
 
+	extensions "k8s.io/api/apps/v1beta2"
```
Comment (luxas) [inline cmd/kubeadm/app/phases/upgrade/health.go]: Should be apps now

**Review by luxas — COMMENTED** [review]
The `extensions` import should be renamed to `apps` now everywhere.

Otherwise, LGTM. Thanks!

**Comment by sbezverk** [comment]
@luxas, completed final cleaning... hopefully it will look good to you ;)

**Review by luxas — APPROVED** [review]
Thanks :tada:!
/lgtm

**Pull request merged**